provider/pkg/provider/provider.go

```diff
@@ -754,7 +754,17 @@ func (p *cfnProvider) Diff(ctx context.Context, req *pulumirpc.DiffRequest) (*pu
 	label := fmt.Sprintf("%s.Diff(%s)", p.name, urn)
 	glog.V(9).Infof("%s executing", label)
 
-	diff, err := p.diffState(req.GetOlds(), req.GetNews(), label)
+	newInputs, err := plugin.UnmarshalProperties(req.GetNews(), plugin.MarshalOptions{
+		Label:        fmt.Sprintf("%s.properties", label),
+		KeepUnknowns: true,
+		RejectAssets: true,
```

Comment:
Never use assets for lambda code?

Reply:
Not currently - Asset support is still a WIP PR which will change these settings everywhere.